plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/data/MicrosoftOfficeApplicationEnum.java

```diff
@@ -0,0 +1,63 @@
+/*********************************************************************
+ * Copyright (c) 2024 Boeing
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Boeing - initial API and implementation
+ **********************************************************************/
+package org.eclipse.osee.framework.core.data;
+
+/**
+ * Enum representing Microsoft Office applications.
+ *
+ * @author Jaden W. Puckett
+ */
+public enum MicrosoftOfficeApplicationEnum {
```

Comment:
Consider switching to OseeEnum as it will serialize correctly.  Also has some useful methods.  Probably don't need to switch now unless you want, but keep in mind for future.

Reply:
Normal enums serialize fine. We use them on MIM.